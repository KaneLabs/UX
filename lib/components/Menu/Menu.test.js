var _interopRequireDefault=require("@babel/runtime/helpers/interopRequireDefault");var _react=_interopRequireDefault(require("react"));var _shallow=_interopRequireDefault(require("react-test-renderer/shallow"));var _Menu=_interopRequireDefault(require("./Menu"));var _this=this,_jsxFileName="/Users/kane/Lab/Eros/src/components/Menu/Menu.test.js";var renderer=new _shallow.default();test('Menu Renders',function(){var tree=renderer.render(_react.default.createElement(_Menu.default,{__self:_this,__source:{fileName:_jsxFileName,lineNumber:8,columnNumber:32}}));expect(tree).toMatchSnapshot();});