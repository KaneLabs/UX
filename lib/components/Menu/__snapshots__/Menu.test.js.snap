// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu Renders 1`] = `
<View>
  <View
    onLayout={[Function]}
    style={
      Object {
        "alignSelf": "flex-start",
        "width": "auto",
      }
    }
  />
  <Modal
    animationType="none"
    onLayout={[Function]}
    transparent={true}
    visible={false}
  >
    <ForwardRef(AnimatedComponentWrapper)
      style={
        Array [
          40,
          Object {
            "height": 0,
            "opacity": 0,
            "top": 0,
            "width": 0,
          },
          undefined,
        ]
      }
      useNativeDriver={false}
    >
      <View
        onLayout={[Function]}
        style={
          Array [
            41,
            Object {
              "width": "auto",
            },
            undefined,
          ]
        }
      >
        <ScrollView
          contentContainerStyle={
            Array [
              Object {
                "paddingVertical": 8,
              },
              undefined,
            ]
          }
        />
      </View>
    </ForwardRef(AnimatedComponentWrapper)>
  </Modal>
</View>
`;
